themes/material/default/selectidp-links.php

```diff
@@ -149,6 +149,9 @@ function clickedAnyway(idpName) {
         align-items: center;
         justify-content: space-around;
     }
+    .mdl-button {
```

Comment:
It might be clearer to focus the rule specifically on the `logo-caption` CSS class, but either way will work. 👍 
```suggestion
    .logo-caption {
```